File: fitrbackend/src/main/java/com/example/fitrbackend/dto/PlanDayResponse.java
package com.example.fitrbackend.dto;

import lombok.Getter;

@Getter
public class PlanDayResponse {

        private final long id;
        private final long workout_plan_id;
        private final int dayNumber;
        private final String name;

        public PlanDayResponse(long id, long workout_plan_id, int dayNumber, String name) {
            this.id = id;
            this.workout_plan_id = workout_plan_id;
            this.dayNumber = dayNumber;
            this.name = name;
        }
}

File: fitrfrontend/fitrfrontend/API/PlanDayResponse.swift
//
//  PlanDayResponse.swift
//  fitrfrontend
//
//  Created by Ambrose Blay on 2/17/26.
//

import Foundation

struct PlanDayResponse: Codable, Identifiable {
    let id: Int64
    let workoutPlanId: Int64
    let dayNumber: Int
    let name: String
    
    enum CodingKeys: String, CodingKey {
        case id
        case workoutPlanId = "workout_plan_id"
        case dayNumber
        case name
    }
}
